.github/workflows/ci-new.yml

```diff
@@ -273,24 +274,8 @@ jobs:
             -shuffle=on \
             -timeout='10m' \
             ./test/integ-new/...
-
-  cleanup:
-    if: '${{ always() }}'
-    needs: ['integration']
-    permissions:
-      contents: 'read'
-      id-token: 'write'
-    runs-on: 'ubuntu-latest'
-    steps:
-      - name: 'Checkout'
-        uses: 'actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b' # ratchet:actions/checkout@v3
-      - id: 'auth'
-        name: 'Authenticate to Google Cloud'
-        uses: 'google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033' # ratchet:google-github-actions/auth@v1
-        with:
-          workload_identity_provider: '${{ env.WIF_PROVIDER }}'
-          service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
-      - id: 'delete_kms_key'
+      - name: "clean up kms key"
```

Comment:
Cloud you also add comment that we cannot delete key, therefore we are destroying key versions so that we are not billed, and add the reference links?